terraform/user_data.sh

```
@@ -0,0 +1,106 @@
+
+aws ssm get-parameter \
+  --name "${ssm_env_parameter}" \
+  --with-decryption \
+  --query "Parameter.Value" \
+  --output text \
+  --region "${aws_region}" \
+  > /tmp/genie.env
+
+# --- Clone repo and deploy ---
+cd /home/ubuntu
+git clone "${github_repo}" genie_nhs_website
```

Shellcheck (0.11.0)

[warning] github_repo is referenced but not assigned.

(SC2154)